experimental/segmenter/src/lstm.rs

```diff
@@ -132,4 +247,37 @@ mod tests {
         let breaks = get_line_break_utf16(&text);
         assert_eq!(breaks, None, "Thai test");
     }
+
+    #[test]
+    fn burmese_word_break() {
+        const TEST_STR: &str = "မြန်မာဘာသာစကား";
+
+        let breaks = get_line_break_utf8(TEST_STR);
+        assert_eq!(breaks.unwrap(), [12, 18, 30], "Burmese test");
```

Comment:
I cannot read Burmese or Thai, so I don't have the ability to verify these break opportunity.

Question: how did you get the data? Are these break opportunities the same as ICU4C break iterator would have generated?

Reply:
This is from wikipedia. This word is "Burmese Language" in Burmese. I will add comment for different of ICU4C result.